docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+
+{{<imgproc src="/tutorials/flutter-app/project-name.png" resize="1200x" style="max-width:800px" declared-dimensions="true" alt="Entering the name for the project in VS Code: 'smart_machine_app'.">}}
+
+Flutter now creates your project folder for you and VS Code opens it.
+
+In the next steps, you'll overwrite the contents of three files with a basic scaffold of the app.
+
+### Copy and paste the initial app
+
+1. In the left pane of VS Code, make sure that **Explorer** is selected, and open the <file>pubspec.yaml</file> file.
+   This file specifies basic information about your app, such as its current version, its dependencies, and the assets with which it will ship.
+
+2. Replace the contents of your <file>pubspec.yaml</file> with the following:
+
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   name: smart_machine_app
+   description: "A new Flutter project."
+
+   publish_to: "none" # Remove this line if you wish to publish to pub.dev
+
+   version: 1.0.0+1
+
+   environment:
+     sdk: ">=3.2.3 <4.0.0"
+
+   dependencies:
+     flutter:
+       sdk: flutter
+     flutter_dotenv: ^5.1.0
+     image: ^4.0.17
+     cupertino_icons: ^1.0.2
+     viam_sdk: ^0.0.9
+
+   dev_dependencies:
+     flutter_test:
+       sdk: flutter
+
+     flutter_lints: ^2.0.0
+
+   flutter:
+     uses-material-design: true
+   ```
+
+   {{% alert title="Note" color="note" %}}
+
+   If you named your app something other than `smart_machine_app`, you need to change the `name` value in the first line of the <file>pubspec.yaml</file> file to the name you gave your app during setup.
+
+   {{% /alert %}}
+
+   3. Next, open another configuration file in the project, <file>analysis_options.yaml</file>.
+      This file determines how strict Flutter should be when analyzing your code.
+      Since this is your first foray into Flutter, you're telling the analyzer to take it easy.
+      You can always tune this later.
+      In fact, as you get closer to publishing an actual production app, you will almost certainly want to make the analyzer stricter than this.
+
+   4. Replace its contents with the following:
+
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   include: package:flutter_lints/flutter.yaml
+
+   linter:
+     rules:
+       avoid_print: false
+       prefer_const_constructors_in_immutables: false
+       prefer_const_constructors: false
+       prefer_const_literals_to_create_immutables: false
+       prefer_final_fields: false
+       unnecessary_breaks: true
+       use_key_in_widget_constructors: false
+   ```
+
+### Configure iOS-specific code
+
+Now you'll update some configurations in the iOS-specific code to support the [Viam Flutter SDK](https://flutter.viam.dev/).
+
+1. Open <file>ios/Podfile</file>.
+   If Podfile does not exist in that directory, you can generate it by running `flutter pub get` in the root directory of your app.
```

Comment:
nit: consider where to run:

```suggestion
   If Podfile does not exist in that directory, you can generate it by running `flutter pub get` in the root directory of your app from the [integrated VS Code terminal](https://code.visualstudio.com/docs/terminal/basics).
```

(actual link is likely overkill, feel free to omit!)

Reply:
Can't they run this command from the root of the directory from another terminal?